packages/@uppy/remote-sources/src/index.js

```diff
@@ -12,7 +12,8 @@ import Zoom from '@uppy/zoom'
 
 import packageJson from '../package.json'
 
-const availablePlugins = [
+const availablePlugins = {
+  __proto__: null,
```

Comment:
:thinking: why?

Reply:
To avoid weird behavior if the user supplies `hasOwnProperty` as pluginId.

Reply:
Would be nice to add that as a comment

Reply:
```suggestion
  // Using a null-prototype object to avoid prototype pollution.
  __proto__: null,
```